cadc-vos-server/src/main/java/org/opencadc/vospace/server/NodePersistence.java

```diff
@@ -131,6 +134,27 @@ public interface NodePersistence {
      */
     public Set<URI> getImmutableProps();
 
+    /**
+     * Get the default value of a Node Property.  This will likely be used by create operations when values are missing.
+     * @param propertyKey   The URI of the property key to get the default value for.
+     * @return  String default value for the given property key, or null if no default value is set.
+     */
+    String getDefaultPropertyValue(final URI propertyKey);
```

Comment:
Seeing how the implementation provides the defaults (config and this method), then the library gets this and carefully injects the default... I think we got this wrong.

The alternative is that the FileSystemNodePersistence.put(...) would evaluate whether default propetries are applicable and add them. That would leave "default quota" a cavern-specific thing with no extra burden on the library or other implementations.

So, I think we remove this from the interface and move that code from library CreateAction into cavern's FileSystemNodePersistence.put(...)

Reply:
Done.